src/models/shared/locations/birthplace.js

```diff
@@ -1,10 +1,32 @@
 import locationModel from '../location'
+import { isPO, isUS, isInternational } from 'helpers/location'
```

Comment:
it looks like only `isInternational` is used here so the other two imports are unnecessary

Reply:
Removed